Client/Frontend/Widgets/ThumbnailCell.swift

```diff
@@ -172,6 +173,20 @@ class ThumbnailCell: UICollectionViewCell {
         return backgroundImage
     }()
 
+    lazy var selectedOverlay: UIView = {
+        let selectedOverlay = UIView()
+        selectedOverlay.backgroundColor = ThumbnailCellUX.SelectedOverlayColor
+        selectedOverlay.hidden = true
+        return selectedOverlay
+    }()
+
+    override var selected: Bool {
+        didSet {
+            super.selected = selected
+            self.selectedOverlay.hidden = (selected) ? false : true
```

Comment:
Good catch - `!selected` would be preferred instead of using the ternary operator to return booleans.